### PR TITLE
Cmor 3.4.0

### PR DESCRIPTION
**Pull request by mauzey1**
Prepare for release of CMOR 3.4.0

### CODE REVIEW

File: Tables/CMIP6_AERhr.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
         "table_id": "Table AERhr", 
         "realm": "aerosol", 
-        "table_date": "13 December 2018", 
+        "table_date": "14 December 2018", 
         "missing_value": "1e20", 
         "int_missing_value": "-999", 
         "product": "model-output", 
-        "approx_interval": "1.0", 
-        "generic_levels": "0.017361", 
+        "approx_interval": "0.041667",
```
Comment (durack1) [inline Tables/CMIP6_AERhr.json]: @taylor13 will this cause problems?

File: Tables/CMIP6_CFsubhr.json [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+        "cmor_version": "3.4", 
         "table_id": "Table CFsubhr", 
         "realm": "atmos", 
-        "table_date": "13 December 2018", 
+        "table_date": "14 December 2018", 
         "missing_value": "1e20", 
         "int_missing_value": "-999", 
         "product": "model-output", 
         "approx_interval": "0.017361", 
+        "approx_interval_error": "0.90",
```
Comment (durack1) [inline Tables/CMIP6_CFsubhr.json]: @taylor13 same here - problem causing?

**Comment by mauzey1** [comment]
I will restart the tests when nightly is done building.

File: Tables/CMIP6_grids.json [hunk 3]
```diff
@@ -45,7 +45,7 @@
             "standard_name": "grid_longitude", 
             "out_name": "rlon", 
             "units": "degrees", 
-            "type": "", 
+            "type": "double",
```
Comment (taylor13) [inline Tables/CMIP6_grids.json]: should be "integer"

**Comment by durack1** [comment]
@taylor13 @mauzey1 I think this is good to merge, right?

**Comment by durack1** [comment]
My mistake, it was already merged, ignore this thread

**Pull request merged**